Makefile

```diff
@@ -308,7 +308,49 @@ go.cachedir:
 go.mod.cachedir:
 	@go env GOMODCACHE
 
-.PHONY: cobertura reviewable submodules fallthrough go.mod.cachedir go.cachedir run crds.clean $(TERRAFORM_PROVIDER_SCHEMA)
+DEP_CONSTRAINT ?= >= 0.0.0
+ifeq (-,$(findstring -,$(VERSION)))
+    DEP_CONSTRAINT = >= 0.0.0-0
+endif
+load-pkg: $(UP) build.all
```

Comment:
In a similar spirit, how about replacing `XPKG_REG_ORGS` with a more user friendly name, such as `REPO`?